samples/nrf9160/modem_shell/src/location/location_cmd_utils.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef MOSH_LOCATION_UTILS_H
+#define MOSH_LOCATION_UTILS_H
+
+enum location_cmd_cloud_data_gnss_format { CLOUD_GNSS_FORMAT_PVT, CLOUD_GNSS_FORMAT_NMEA };
+
+int location_cmd_utils_generate_nmea(const struct location_data *location_data, char *nmea_buf,
```

Comment:
This function is currently only used inside `location_cmd_utils.c`, so it isn't needed in the header file (and it could be static).

Reply:
Fixed